src/main/java/teammates/ui/pagedata/InstructorCoursesPageData.java

```diff
@@ -100,6 +100,12 @@ private ArchivedCoursesTable convertToArchivedCoursesTable(List<CourseAttributes
 
             ArchivedCoursesTableRow row = new ArchivedCoursesTableRow(SanitizationHelper.sanitizeForHtml(course.getId()),
                                                                       SanitizationHelper.sanitizeForHtml(course.getName()),
+                                                                      SanitizationHelper.sanitizeForHtml(
+                                                                              course.getCreatedAtDateString()),
+                                                                      SanitizationHelper.sanitizeForHtml(
+                                                                              course.getCreatedAtDateStamp()),
+                                                                      SanitizationHelper.sanitizeForHtml(
+                                                                              course.getCreatedAtFullDateTimeString()),
```

Comment:
I believe these 3 fields don't need sanitisation as they are safe strings not input by the user but generated by the system from a date object.